bot/interact.go

```diff
@@ -65,8 +66,21 @@ func interactAPIHandler(botContext cmd.BotContext) http.HandlerFunc {
 					validationErrors["job_link"] = "The link to the job spec is invalid"
 				}
 
-				if strings.Contains(strings.ToLower(message.Submission["max_salary"]), "stock") {
-					validationErrors["max_salary"] = "The Salary Max field cannot contain extras like mentions to Stock Options."
+				maxSalary, err := strconv.Atoi(strings.TrimSpace(message.Submission["max_salary"]))
+				if err != nil || maxSalary == 0 {
+					validationErrors["max_salary"] = "The Salary Max field should be a non-zero numeric value."
+				}
+
+				var minSalary int
+				if minSalaryStr := strings.TrimSpace(message.Submission["max_salary"]); minSalaryStr != "" {
```

Comment:
```suggestion
				if minSalaryStr := strings.TrimSpace(message.Submission["min_salary"]); minSalaryStr != "" {
```

Reply:
Yeah, there was a typo. I discovered while testing locally (we have a channel for testing candebot)

Reply:
But super good catch btw :)